### ISSUE TITLE
Feature Request: Dual Monitor Support for Touchscreens

### ISSUE THREAD

**Issue by RazGAWD** [Arisotura/melonDS#1399]
Sorry if a duplicate. Was searching pull requests. Maybe the third time I have been on GitHub.

**Comment by poudink** [comment]
yeah this is a duplicate https://github.com/Arisotura/melonDS/issues/1379 https://github.com/Arisotura/melonDS/issues/1112
also a feature listed in the readme's todo list.
